sql/config/daily_regression_test_exclude_list_compatibility/patch_files/jdbc_sql_9.3.0_D_patch

```diff
@@ -67165,132 +67069,6 @@ Index: _24_aprium_qa/_05_non-ASCII/answers/bug_bts_10624.answer
--7     æ¢¨è?±     6     æ¢¨è?±     6     Æ¢¨È?±     6     
--8     æ¡?æ¢?     6     æ¡?æ¢?     6     Æ¡?Æ¢?     6     
--9     ç§?å­¦     6     ç§?å­¦     6     Ç§?Å­¦     6     
--10     å¤¹ç«¹æ¡?     9     å¤¹ç«¹æ¡?     9     Å¤¹Ç«¹Æ¡?     9     
--103     ä½ ä»¬     6     ä½ ä»¬     6     Ä½ Ä»¬     6     
--104     è¿?æ?¥     6     è¿?æ?¥     6     È¿?Æ?¥     6     
--105     ç§?æµ·æ£      9     ç§?æµ·æ£      9     Ç§?ÆΜ·Æ£      9     
--106     %æ°´ä»?     7     %æ°´ä»?     7     %Æ°´Ä»?     7     
--107     _æ¢¨è?±     7     _æ¢¨è?±     7     _Æ¢¨È?±     7     
--109     ?ç§?å­¦     7     ?ç§?å­¦     7     ?Ç§?Å­¦     7     
--110     *å¤¹ç«¹æ¡?     10     *å¤¹ç«¹æ¡?     10     *Å¤¹Ç«¹Æ¡?     10     
--203     ä½ ä»¬     6     ä½ ä»¬     6     Ä½ Ä»¬     6     
--205     ç¢§æ¡?     6     ç¢§æ¡?     6     Ç¢§Æ¡?     6     
--303     ä½ ä»¬.æ??ä»¬     13     ä½ ä»¬.æ??ä»¬     13     Ä½ Ä»¬.Æ??Ä»¬     13     
--403     ä½ ä»¬ ä»?ä»¬     13     ä½ ä»¬ ä»?ä»¬     13     Ä½ Ä»¬ Ä»?Ä»¬     13     
--
--===================================================
--id    name    octet_length(name)    lower(name)    octet_length( lower(name))    upper(name)    octet_length( upper(name))    
--1     é¸¢å°¾è?±     17     é¸¢å°¾è?±     17     É¸¢Å°¾È?±     17     
--2     ä½ ä»¬     12     ä½ ä»¬     12     Ä½ Ä»¬     12     
--3     ä½ ä»¬     12     ä½ ä»¬     12     Ä½ Ä»¬     12     
--4     ä»?ä»¬     11     ä»?ä»¬     11     Ä»?Ä»¬     11     
--5     æ??ä»¬     10     æ??ä»¬     10     Æ??Ä»¬     10     
--6     æ°´ä»?     11     æ°´ä»?     11     Æ°´Ä»?     11     
--7     æ¢¨è?±     11     æ¢¨è?±     11     Æ¢¨È?±     11     
--8     æ¡?æ¢?     10     æ¡?æ¢?     10     Æ¡?Æ¢?     10     
--9     ç§?å­¦     11     ç§?å­¦     11     Ç§?Å­¦     11     
--10     å¤¹ç«¹æ¡?     17     å¤¹ç«¹æ¡?     17     Å¤¹Ç«¹Æ¡?     17     
--103     ä½ ä»¬     12     ä½ ä»¬     12     Ä½ Ä»¬     12     
--104     è¿?æ?¥     10     è¿?æ?¥     10     È¿?Æ?¥     10     
--105     ç§?æµ·æ£      17     ç§?æµ·æ£      17     Ç§?ÆΜ·Æ£      17     
--106     %æ°´ä»?     12     %æ°´ä»?     12     %Æ°´Ä»?     12     
--107     _æ¢¨è?±     12     _æ¢¨è?±     12     _Æ¢¨È?±     12     
--109     ?ç§?å­¦     12     ?ç§?å­¦     12     ?Ç§?Å­¦     12     
--110     *å¤¹ç«¹æ¡?     18     *å¤¹ç«¹æ¡?     18     *Å¤¹Ç«¹Æ¡?     18     
--203     ä½ ä»¬     12     ä½ ä»¬     12     Ä½ Ä»¬     12     
--205     ç¢§æ¡?     11     ç¢§æ¡?     11     Ç¢§Æ¡?     11     
--303     ä½ ä»¬.æ??ä»¬     23     ä½ ä»¬.æ??ä»¬     23     Ä½ Ä»¬.Æ??Ä»¬     23     
--403     ä½ ä»¬ ä»?ä»¬     24     ä½ ä»¬ ä»?ä»¬     24     Ä½ Ä»¬ Ä»?Ä»¬     24     
--
-+Error:-493
- ===================================================
--0
-+Error:-493
- ===================================================
--0
-+Error:-494
-+===================================================
-+Error:-839
- ===================================================
- 'to_date( , DAY )'    
- to_date( , DAY )     
```

Comment:
Many patch contents have been lost.
Is this no problem?

Reply:
The former modification was wrong. These cases have to be executed fine as this commit.
But an error occurs because 'make_locale=no' was configured while modifying the former commit.
So, deleting the contents is the right conclusion.